packages/app/src/main.ts

```diff
@@ -10,6 +10,7 @@ import { AppModule } from './app/app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  app.enableCors();
```

Comment:
Do you need this?

Reply:
Yes. The UI and backend run on different ports, the browser considers that cross-origin.